### PR TITLE
Fix behavior of is_low_contrast for boolean inputs

### PR DESCRIPTION
**Pull request by grlee77**
## Description

This PR rreplaces #5247 which had been inactive for a few months. A simple test function was added.

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: skimage/exposure/exposure.py [hunk 1]
```diff
@@ -625,6 +625,11 @@ def is_low_contrast(image, fraction_threshold=0.05, lower_percentile=1,
     out : bool
         True when the image is determined to be low contrast.
 
+    Notes
+    -----
+    For boolean images, this function returns False only if all values are
+    the same (the method, threshold and percentile arguments are ignored).
```
Comment (mkcor) [inline skimage/exposure/exposure.py:631]:
```suggestion
    the same (the method, threshold, and percentile arguments are ignored).
```
(yes, I love the Oxford comma)

File: skimage/exposure/tests/test_exposure.py [hunk 2]
```diff
@@ -748,6 +748,14 @@ def test_is_low_contrast():
     assert not exposure.is_low_contrast(image, upper_percentile=100)
 
 
+def test_is_low_contrast_boolean():
+    image = np.zeros((8, 8))
```
Comment (mkcor) [inline skimage/exposure/tests/test_exposure.py:752]:
```suggestion
    image = np.zeros((8, 8), dtype=bool)
```

**Review by mkcor — COMMENTED** [review]
Thanks for rescuing #5467! I suggested adding the key part of the test :wink:

File: skimage/exposure/tests/test_exposure.py (hunk 2 above)
Comment (grlee77) [reply skimage/exposure/tests/test_exposure.py:752]: Definitely need `bool` here! This actually uncovered a copy paste error in the new code above!

**Review by mkcor — APPROVED** [review]

**Comment by alexdesiqueira** [comment]
Thank you @grlee77!

**Pull request merged**